charter.md

```diff
@@ -63,6 +63,30 @@ are able to proceed with a meeting.
 * Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
 * Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
 
+## Elections
+
+Every year, the Steering Committee holds an election for open seats with SIG 
+Contributor Experience owning the process. Our [election policy document](https://git.k8s.io/steering/elections.md) 
+covers the details for how this works.
+### Vacancies and Removal
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+#### Removal
+
+A Steering Committee member may be removed by a two-thirds vote of the other 
```

Comment:
Are there any limitations or guidance for when the steering committee should hold a removal vote?  Any defined threshold for committee members to consider when deciding to remove a person from office?

"high crimes and misdemeanors" is vague and defined separately, but also makes it clear that the judgement about impeachment (equivalent to removal?) is about whether a criminal act has been committed versus a distasteful act or simply being unpopular among peers.

Reply:
> Are there any limitations or guidance for when the steering committee should hold a removal vote? Any defined threshold for committee members to consider when deciding to remove a person from office?
> 
> "high crimes and misdemeanors" is vague and defined separately, but also makes it clear that the judgement about impeachment (equivalent to removal?) is about whether a criminal act has been committed versus a distasteful act or simply being unpopular among peers.

While I don't disagree that there should be some type of guidance or limitations, I think comparing it to impeachment is a bit excessive here...  Setting the bar at criminal acts is way to low for a body that is supposed to `Define, evolve, and defend the non-technical vision / mission and the values of the project`. I think we *must* expect the Steering Committee to hold itself accountable to the community by adhering to those defined values (https://github.com/kubernetes/community/blob/master/values.md). We also shouldn't expect that a sitting member of the Steering committee be immune from things like Code of Conduct violations and enforcement, would we?

Reply:
> While I don't disagree that there should be some type of guidance or limitations, I think comparing it to impeachment is a bit excessive here... Setting the bar at criminal acts is way to low for a body that is supposed to Define, evolve, and defend the non-technical vision / mission and the values of the project. I think we must expect the Steering Committee to hold itself accountable to the community by adhering to those defined values (https://github.com/kubernetes/community/blob/master/values.md).

Agreed

> We also shouldn't expect that a sitting member of the Steering committee be immune from things like Code of Conduct violations and enforcement, would we?

Not at all. The list at https://github.com/kubernetes/community/blob/master/committee-code-of-conduct/charter.md#removal could be useful prior art in this area to consider.

Reply:
As I mentioned https://github.com/kubernetes/steering/pull/248#discussion_r911317290, the intent is not to address code of conduct issues, but if, for any reason, the committee has no confidence in a member and feels that member is unfit to serve.

Reply:
Members of the Steering Committee have previously [raised issue with the idea of being beholden to the CoCC in the same way other members of the project are.](https://github.com/kubernetes/community/pull/6247) 

I do not agree with the objections raised in the linked PR, but I do respect Steering's right to self determine their processes.

As a result I do not think Steering needs to consider any prior art in the Code of Conduct Committee's processes. We are independent bodies and are both free to set our own process independent of one another.

I see no problem with the wording or process presented here by SC. 👍, +1, /lgtm.

Reply:
> when a member has no confidence in the leadership of the individual in question which could encompass most of your examples. do you think that would make it clear?

To a skeptical person, "no confidence" reads as, "for any reason someone wants" versus a guideline that indicates boundaries between taste, preference, opinion, and behavior.  I think a guideline that focuses on behavior and actions will be easiest to explain to observers of the process and easiest for the committee to consider when casting their votes.

Reply:
@deads2k thanks for the feedback! we'll see what we can do.

Reply:
I think the more transparent the process, the less important this guideline is (and vice versa).

Reply:
+1 @lavalamp